gtk/src/videohandler.c

```diff
@@ -64,6 +64,21 @@ int ghb_set_video_preset(GhbValue *settings, int encoder, const char * preset)
     return result;
 }
 
+void
+ghb_update_multipass(signal_user_data_t *ud)
+{
+    GtkWidget *multi_pass = GTK_WIDGET(ghb_builder_widget("VideoMultiPassBox"));
+    GtkWidget *turbo_multi_pass = GTK_WIDGET(ghb_builder_widget("VideoTurboMultiPass"));
```

Comment:
There's no need to wrap these in GTK_WIDGET() macros as the function returns a GtkWidget pointer.

Reply:
Done